packages/wonder-blocks-tooltip/components/tooltip-bubble.md

```diff
@@ -0,0 +1,192 @@
+This is an internal component that we use to render the stuff that appears when a tooltip shows.
+
+Note that without explicit positioning, the arrow will not be centered.
+
+### Placement top
+
+```jsx
+const {View} = require("@khanacademy/wonder-blocks-core");
+
+const popperProps = {
+    style: {
+        position: "absolute",
+        top: 0,
+        left: 0,
+    },
+    placement: "top",
+    ref: () => {},
+    scheduleUpdate: () => {},
+    arrowProps: {
+        style: {
+            left: 0,
+            top: 0,
+        },
+        ref: () => {},
+    },
+    outOfBoundaries: false,
+};
+
+<View style={{height: 50, flexDirection: "column"}}>
+    <TooltipBubble popperProps={popperProps}>
+        <TooltipContent>I'm on the top!</TooltipContent>
+    </TooltipBubble>
+</View>
+```
+
+### Placement right
+
+```jsx
+const {View} = require("@khanacademy/wonder-blocks-core");
+
+const popperProps = {
+    style: {
+        position: "absolute",
+        top: 0,
+        left: 0,
+    },
+    placement: "right",
+    ref: () => {},
+    scheduleUpdate: () => {},
+    arrowProps: {
+        style: {
+            left: 0,
+            top: 0,
+        },
+        ref: () => {},
+    },
+    outOfBoundaries: false,
+};
+
+<View style={{height: 50, flexDirection: "row-reverse", justifyContent: "flex-end"}}>
```

Comment:
Using the `style` prop to override `flexDirection` won't work b/c we set `!important` on the classes applied to the underlying `div`.  In this case `flexDirection` has no effect anyways b/c there's only a single child.  Please  remove all `flexDirection`s in the examples.

Reply:
It is definitely working. Can you tell me how it won't work?

Reply:
Oh wait, I got confused. It's working in the `TooltipArrow` examples. I copied this from there. I don't need that in this case, so I should remove it.